src/canvas-extractor/edfi_canvas_extractor/client_graphql.py

```diff
@@ -9,23 +9,22 @@
 from pandas import DataFrame
 from typing import Dict, List, Tuple
 
-from canvasapi.course import Course
-from canvasapi.enrollment import Enrollment
-from canvasapi.section import Section
-from canvasapi.user import User
+from canvasapi.submission import Submission
```

Comment:
QUESTION: Is there a reason we are still using `canvasapi` for `Submission` instead of `Dict[str, str]`?

Reply:
Remove all the canvasapi dependecies